### ISSUE TITLE
[request] wxWidgets/3.2.1

### ISSUE THREAD

**Issue by hweemiin** [conan-io/conan-center-index#20092]
### Package Name/Version

wxwidgets/3.2.1

### Changelog

https://raw.githubusercontent.com/wxWidgets/wxWidgets/v3.2.1/docs/changes.txt

### Context about the new update

The above-mentioned version is newly released by the upstream project and not yet available as a recipe.
Please add this version.

**Comment by samuel-emrys** [comment]
@hweemiin is there a reason you've raised your PR for bincrafters instead of CCI?

**Comment by hweemiin** [comment]
hmm, I found previous wxWidgets recipe on bincrafters first, then I click on the link of "Request new package" on bincrafters page and it send me to CCI. Not sure what's the relationship between the two, is bincrafters obsolete and being replaced by CCI?

**Comment by samuel-emrys** [comment]
I'm not sure what the current state of bincrafters is, but it's only had 1 commit in the past 2 years. Maybe @Croydon can provide some context here.

My understanding is that it hosts recipes that can't be hosted on CCI for whatever reason. [It looks like the reasons it hasn't been on CCI historically may be largely resolved](https://github.com/bincrafters/community/issues/1441#issuecomment-1148478883) given that many of the system dependencies for wxwidgets appear to now be available as cci recipes

**Comment by luizfeldmann** [comment]
Linking the [recipe](https://github.com/bincrafters/conan-wxwidgets/tree/stable/3.1.3) from Bincrafters.

[This](https://github.com/microsoft/vcpkg/blob/master/ports/wxwidgets/vcpkg.json) is a convenient place to see what are the dependencies.

**Comment by samuel-emrys** [comment]
> Linking the recipe from Bincrafters

This is an old/deprecated recipe. The current recipe is here https://github.com/bincrafters/community/blob/main/recipes/wxwidgets/all/conanfile.py

**Comment by luizfeldmann** [comment]
Can CCI 'steal' working recipes from there ?

**Comment by samuel-emrys** [comment]
> Can CCI 'steal' working recipes from there ?

The bincrafters recipes appear to be MIT licensed, so I don't see any problems using those as a base to build a CCI recipe on, however, as above:

> My understanding is that it [bincrafters] hosts recipes that can't be hosted on CCI for whatever reason. [It looks like the reasons it hasn't been on CCI historically may be largely resolved](https://github.com/bincrafters/community/issues/1441#issuecomment-1148478883) given that many of the system dependencies for wxwidgets appear to now be available as cci recipes

So it's not obvious to me that they would be suitable out of the box.

**Comment by uilianries** [comment]
> Can CCI 'steal' working recipes from there ?

Yes. Bincrafters is under MIT license. (I'm one of Bincrafters' maintainers too)

> My understanding is that it hosts recipes that can't be hosted on CCI for whatever reason. https://github.com/bincrafters/community/issues/1441#issuecomment-1148478883 given that many of the system dependencies for wxwidgets appear to now be available as cci recipes

True. The main project that was not acceptable in CCI, but want in Bincrafters is the modular boost, where each boost project is a Conan package. It would required a prepared CI only for it, to solve the graph, rebuild everything ... something not inclined to be accepted here.

**Comment by herring-swe** [comment]
I made a 3.2.3 package for conan 2 a while ago which works on Windows and Linux.
Actually tested today on conan v1 (1.62) and it works there as well.
It's based on bincrafters 3.1.5 recipe but heavily modified so it finds built libraries via cmake installed config files and expose them as components,.
Maybe something of that can be merged into your recipe @gmeeker.

**Comment by Croydon** [comment]
Done by #21353

**Comment by uilianries** [comment]
Hello! This issue is now fixed by #27391

Please, update your local copy to the latest version available:

    conan install -r conancenter --update --requires=wxwidgets/3.2.8

Regards!
